nixos/modules/services/web-apps/youtrack.md

```diff
@@ -0,0 +1,27 @@
+# YouTrack {#module-services-youtrack}
+
+YouTrack is a browser-based bug tracker, issue tracking system and project management software.
+
+## Installation {#module-services-youtrack-installation}
+
+YouTrack exposes a web GUI installer on first login.
+You need a token to access it.
+You can find this token in the log of the `youtrack` service. The log line looks like
+```
+* JetBrains YouTrack 2023.3 Configuration Wizard will be available on [http://127.0.0.1:8090/?wizard_token=somelongtoken] after start
+```
+
+## Upgrade from 2022.3 to 2023.x {#module-services-youtrack-upgrade-2022_3-2023_1}
```

Comment:
I had issues with YouTrack DB migration while updating from `2022.3` to `2023.3`.
It was suggested to provide the `-J-Dexodus.entityStore.refactoring.forceAll=true` JVM option for the first YT startup after the upgrade. Perhaps, it's worth mentioning to save someone else time in the future.
However, I couldn't reproduce this on a small YT instance with a free license, so this might only affect some old-ish big YT installations.